### DISCUSSION TITLE
PSC proposal: yajo in Procurement & Purchase

### DISCUSSION THREAD

**Discussion by yajo** [OCA/odoo-community.org#109] [Polls]
I'd like to propose myself as PSC in the Procurement & Purchase category.

You won't see much of my personal PRs there, but you can see many from Moduon, and I review most of them. I'd like to be able to work with more agility.

Thanks!

**Comment by rousseldenis** [comment]
Hi, +1 for me.

the problem with such vote, we cannot see if there are approvals from PSC members.

**Reply by yajo** [reply]
Ah interesting... I guess they can hit the 👍🏼 icon then. That way we know who approved. Or add a comment.

Sorry, I was just trying, but obviously these votes are not good for PSC declaring. Let me try to remove the poll.

**Reply by yajo** [reply]
Well, I cannot update it.

**Reply by yajo** [reply]
I modified the poll so it's empty now.
